roles/etcd/defaults/main.yml

```diff
@@ -1,14 +1,53 @@
 ---
 # role variable for the etcd service
 
+etcd_version: v2.3.8
+etcd_cluster_name: contiv
+
+etcd_proxy: "{{ etcd_proxies_group in groups
+                 and inventory_hostname in groups[etcd_proxies_group] }}"
```

Comment:
is there any chance `etcd_proxies_group in groups` and not `inventory_hostname in groups[etcd_proxies_group]` ?

Reply:
oh how about etcd_proxies_group in group_names

Reply:
yeah, that's better

Reply:
etcd_proxies_group in groups was to validate i didn't get a bad dereference, otherwise I would have just checked `inventory_hostname in groups[etcd_proxies_group]` . . . if I could do |default(False) that would be cool but i didn't test that out

Reply:
i tried out the etcd_proxies_group in group_names, group_names was not defined, maybe it's ansible 2.3

Reply:
hum, it's a feature since 1.9 ... . anyway, it's not a blocker